.github/workflows/build-riscv64.yml

```diff
@@ -0,0 +1,80 @@
+name: QEMU Build and test riscv64 crate
+
+on:
+  push:
+    branches:
+      - main
+      - dev
+    tags:
+        - '**'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build_crate:
+    name: Build riscv64 crate and run tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 1
+
+    - name: Set up QEMU on x86_64
+      id: qemu
+      uses: docker/setup-qemu-action@v2
+      with:
+        platforms: riscv64
+
+    - name: Build and Test
+      run: |
+        docker run --rm --platform linux/riscv64 \
+          -v ${{ github.workspace }}:/ws --workdir=/ws \
+          chianetwork/ubuntu-22.04-risc-builder:latest \
+          bash -exc '\
+          cargo test --release
+          '
+
+  build_wheels:
+    name: QEMU riscv64 wheel
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 1
+
+    - name: Set up QEMU on x86_64
+      id: qemu
+      uses: docker/setup-qemu-action@v2
+      with:
+        platforms: riscv64
+
+    - name: Build and Test
+      run: |
+        docker run --rm --platform linux/riscv64 \
+          -v ${{ github.workspace }}:/ws --workdir=/ws \
+          chianetwork/ubuntu-22.04-risc-builder:latest \
+          bash -exc '\
+          pyenv global 3.10
+          python -m venv venv && \
+          source ./venv/bin/activate && \
+          pip install --upgrade pip && \
+          pip install --extra-index-url https://pypi.chia.net/simple/ maturin==1.2.3 && \
+          maturin build -i python --release -m wheel/Cargo.toml \
```

Comment:
since you don't run the python tests, building the wheel in release mode will only slow it down. The python tests aren't very demanding, so even running them against a debug build is fine

Reply:
oh, right. you publish the wheel, so it should be in release mode